src/charm.py

```diff
@@ -110,6 +125,19 @@ def _pgbouncer_layer(self) -> Layer:
             },
         }
 
+    def _on_update_status(self, _) -> None:
+        """Update Status hook.
+
+        Uses systemd status to verify pgbouncer is running.
```

Comment:
It is using the availability of the backend relation, right?

Reply:
Whoops, copy-paste error. You're right, pebble should be handling whether pgbouncer is running, so it's more about checking we're connected.